packages/quantic/force-app/main/default/lwc/headlessLoader/headlessLoader.js

```diff
@@ -49,64 +52,92 @@ const executeInitialSearch = debounce(() => {
 /**
  * Returns true if registered components are initialized, false otherwise.
  */
-const areAllComponentsInitialized = () => !window.coveoHeadless.components.find(component => component.initialized === false);
+const areAllComponentsInitialized = () =>
+  !window.coveoHeadless.components.find(
+    (component) => component.initialized === false
+  );
 
 /**
- * Returns the registered component object if it exists. 
- * @param element 
+ * Returns the registered component object if it exists.
+ * @param element
  */
-const getRegisteredComponent = (element) => window.coveoHeadless.components.find((component) => component.element === element);
+const getRegisteredComponent = (element) =>
+  window.coveoHeadless.components.find(
+    (component) => component.element === element
+  );
 
 /**
- * Loads dependencies and returns an initialized Headless engine. 
- * @param element 
+ * Loads dependencies and returns an initialized Headless engine.
+ * @param element
```

Comment:
I assume `element` here and above are HTML elements? You can add a type to them like so:
```js
/*
 * ...
 * @param {HTMLElement} element
 */
```

Reply:
Check on @nathanlb PR for this :) https://github.com/coveo/ui-kit/pull/572

Reply:
TODO: rebase PR on master and remove all JavaScript.